docs/dev/run-cvo-locally.md

```diff
@@ -0,0 +1,51 @@
+# How to Build and Run CVO Locally
+
+This document provides the instructions to build and run the CVO executable locally. Local execution of CVO, for example executing CVO on your development laptop, allows a "quick-turn" test capability whereby the CVO executable can be quickly rebuilt with changes and launched again without the need of starting a new OCP cluster.
+
+## Preconditions
+
+- Build the CVO executable using the standard procedure:
+
+```console
```

Comment:
Nit: I don't think it's worth using the `- ` list for this section when we can use paragraphs.  But if you stick with the list, things like this console block should be four-space indented so they show up under their associated list entry instead of appearing in the main flow after the list entry completes.

Reply:
Changed in c7cdbc3 to use paragraphs